tailwind.config.js

```diff
@@ -1,4 +1,6 @@
-/** @type {import('tailwindcss').Config} */
+/** @type {import("tailwindcss").Config} */
+import colorPallete from "./data-repo/colorPalette.json";
+
```

Comment:
The JS Doc type comment belong to config and not color pallete

```suggestion
import colorPallete from "./data-repo/colorPalette.json";

/** @type {import("tailwindcss").Config} */
```

Reply:
Okay, I need to swap it right ?🤔
and what is the name of file ? colors or themes.json?

Reply:
No changes in file name. The JS Doc type comment belongs to the exported config variable. just place the the type comment for the config. 

https://www.jetbrains.com/help/webstorm/creating-jsdoc-comments.html#ws_js_jsdoc_typescript_syntax